scripts/test_restore_points.sh

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env bash
+
+#
+# Test PostgreSQL restore point recovery for single node (1 instance, regular hypertable)
+# and multi node cluster (4 instances, distributed hypertable).
+#
+# Make sure that single and multi node results match after recovery.
+#
+
+set -e
+set -o pipefail
+
+SN_NAME=${SN_NAME:-sn}
+AN_NAME=${AN_NAME:-an}
+DN1_NAME=${DN1_NAME:-dn1}
+DN2_NAME=${DN2_NAME:-dn2}
+DN3_NAME=${DN3_NAME:-dn3}
+
+HOST=${HOST:-'localhost'}
+SN_PORT=${SN_PORT:-5432}
+AN_PORT=${AN_PORT:-5432}
+DN1_PORT=${DN1_PORT:-5433}
+DN2_PORT=${DN2_PORT:-5434}
+DN3_PORT=${DN3_PORT:-5435}
+
+TEST_DIR=${TEST_DIR:-`pwd`}
+WAL_DIR=${TEST_DIR}/wal
+BASEBACKUP_DIR=${TEST_DIR}/basebackup
+STORAGE_DIR=${TEST_DIR}/storage
+RESULT_DIR=${TEST_DIR}/result
+
+TEST_RESTORE_POINT_1=${TEST_RESTORE_POINT_1:-rp1}
+TEST_RESTORE_POINT_2=${TEST_RESTORE_POINT_2:-rp2}
+TEST_RESTORE_POINT_3=${TEST_RESTORE_POINT_3:-rp3}
+
+function instance_configure()
+{
+	NAME=$1
+	PORT=$2
+	RESTORE_POINT_NAME=$3
+	pg_ctl init -D "${STORAGE_DIR}/${NAME}"
```

Comment:
I would make the script support PG tools in a non-standard path by prefixing like this:

```suggestion
	${PG_BIN}/pg_ctl init -D "${STORAGE_DIR}/${NAME}"
```

Then allow overriding `PG_BIN` above. This applies to all `psql` calls below as well.

Reply:
I guess I can do this, but I believe we don't do that for our other scripts. Personally I would prefer to change `PATH` to point to proper pg directory instead before running this script, if someone need to change the version.